Album/ViewControllers/ProfileControllers/ProfileViewController.m

```diff
@@ -22,71 +22,71 @@ @interface ProfileViewController () <UICollectionViewDataSource, UICollectionVie
-            }
-        }];
-    }
+	PFUser *user = [PFUser currentUser];
+	if(user[@"profileImage"]) {
+		PFFileObject *file = user[@"profileImage"];
+		[self.profileImageView setFile:file];
+		[file getDataInBackgroundWithBlock:^(NSData *imageData, NSError *error) {
+		         if (!error) {
+				 UIImage *image = [UIImage imageWithData:imageData];
+				 [self.profileImageView setImage:image];
+				 self.profileImageView.layer.cornerRadius = self.profileImageView.frame.size.height/2;
+				 self.profileImageView.layer.masksToBounds = YES;
+			 }
+		 }];
+	}
 }
 
 #pragma mark - Navigation
 
 // In a storyboard-based application, you will often want to do a little preparation before navigation
 - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
-    // Get the new view controller using [segue destinationViewController].
-    // Pass the selected object to the new view controller.
-    if([segue.identifier isEqualToString:@"activitiesSegue"]){
-        UINavigationController *navigationController = [segue destinationViewController];
-        ActivitiesViewController *activitiesController = (ActivitiesViewController*)navigationController.topViewController;
-        //        activitiesController.delegate = self; TODO: Add delegate later
-        
-    }else if([segue.identifier isEqualToString:@"addFriendSegue"]) {
-        UINavigationController *navigationController = [segue destinationViewController];
-        AddFriendViewController *addFriendController = (AddFriendViewController*)navigationController.topViewController;
-        //        addFriendController.delegate = self; TODO: Add delegate later
-    }else if([segue.identifier isEqualToString:@"settingsSegue"]) {
-        UINavigationController *navigationController = [segue destinationViewController];
-        SettingsViewController *settingsController = (SettingsViewController*)navigationController.topViewController;
-        //        settingsController.delegate = self; TODO: Add delegate later
-    }
+	// Get the new view controller using [segue destinationViewController].
+	// Pass the selected object to the new view controller.
+	if([segue.identifier isEqualToString:@"activitiesSegue"]) {
+		UINavigationController *navigationController = [segue destinationViewController];
+		ActivitiesViewController *activitiesController = (ActivitiesViewController*)navigationController.topViewController;
+		//        activitiesController.delegate = self; TODO: Add delegate later
+
+	}else if([segue.identifier isEqualToString:@"addFriendSegue"]) {
+		UINavigationController *navigationController = [segue destinationViewController];
+		AddFriendViewController *addFriendController = (AddFriendViewController*)navigationController.topViewController;
+		//        addFriendController.delegate = self; TODO: Add delegate later
+	}else if([segue.identifier isEqualToString:@"settingsSegue"]) {
+		UINavigationController *navigationController = [segue destinationViewController];
```

Comment:
seems like:
```
UINavigationController *navigationController = [segue destinationViewController];
```
is repeated on every if-statement

Reply:
Oops. Refactored. Thank you!